analyses/fusion_filtering/01-fusion-standardization.R

```diff
@@ -0,0 +1,90 @@
+# K. S. Gaonkar 2019
+# Standardizes fusion calls from callers [STARfusion|| Arriba]. The output will have the following columns
```

Comment:
For now they aren't . . I've not decided yet if that is not useful information to capture in the output. @jharenza  what do you think of keeping breakpoint info in the output?

Reply:
@jaclyn-taroni currently it is not retained (before thinking about a package, we were thinking about the results as a supplemental table - we can perhaps break this up into two now), but I think it would be definitely good to retain this, since following annotation/prioritization, having that information (surrounding nucleotides/breakpoint/chr/coordinates/exons) would be useful for sequence/RACE validation, if desired, so people wouldn't have to go back to raw files to find that information.

Reply:
Sounds good 👍  If it stays in what is returned by `standard_fusion`, then the breakpoint steps should get retained + it should be documented here.